src/antares/model/link.py

```diff
@@ -112,6 +123,12 @@ class LinkUi(DefaultLinkUi):
 
 
 class LinkUiLocal(DefaultLinkUi):
+    link_style: LinkStyle = Field(default=LinkStyle.PLAIN, alias="link-style")
+    link_width: float = Field(default=1, alias="link-width")
+    colorr: int = Field(default=112, alias="colorr")
```

Comment:
Don't need to introduce aliases when it's the same as the python variable

Reply:
Removed next commit